lib/standalone.js

```diff
@@ -134,7 +134,7 @@ async function standalone({
 		const postcssResult = stylelintResult._postcssResult;
 		const returnValue = prepareReturnValue([stylelintResult], maxWarnings, formatterFunction, cwd);
 
-		if (fix && postcssResult && !postcssResult.stylelint.ignored) {
+		if ((fix || config?.fix) && postcssResult && !postcssResult.stylelint.ignored) {
```

Comment:
[suggest] I think the [`??`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing) operator can clear the code, like this:

```suggestion
		const autofix = fix ?? config?.fix ?? false;

		if (autofix && postcssResult && !postcssResult.stylelint.ignored) {
```

Note: We can use this operator since Node.js 14 supports it.